official_samples/K3FSM/language_workbench/org.eclipse.gemoc.example.k3fsm.k3dsa/src/org/eclipse/gemoc/example/k3fsm/k3dsa/k3fsmAspects.xtend

```diff
@@ -4,56 +4,56 @@ import fr.inria.diverse.k3.al.annotationprocessor.Aspect
 import fr.inria.diverse.k3.al.annotationprocessor.InitializeModel
 import fr.inria.diverse.k3.al.annotationprocessor.Main
 import fr.inria.diverse.k3.al.annotationprocessor.Step
+import org.eclipse.emf.common.util.EList
 import org.eclipse.gemoc.example.k3fsm.FSM
 import org.eclipse.gemoc.example.k3fsm.State
 import org.eclipse.gemoc.example.k3fsm.Transition
-import org.eclipse.emf.common.util.EList
-	
+
+import static extension org.eclipse.gemoc.example.k3fsm.k3dsa.StateAspect.*
 import static extension org.eclipse.gemoc.example.k3fsm.k3dsa.TransitionAspect.*
-//import static extension org.eclipse.gemoc.example.k3fsm.k3dsa.FSMAspect.*			
-import static extension org.eclipse.gemoc.example.k3fsm.k3dsa.StateAspect.*			
 
 @Aspect(className=FSM)
 class FSMAspect {
-	@Step 												
-	@InitializeModel									// <1>
```

Comment:
the // <1>  is a directive for the asciidoc generator, it must stay in the file (or find another way to document it)
cf. https://download.eclipse.org/gemoc/docs/nightly/K3FSM-example.html#_add_semantics_operations